### ISSUE TITLE
Use LargeText plugin in Tabular Forms

### ISSUE THREAD

**Issue by pettinger** [martinbnielsen/apex-mbn-large-text#1]
Is there a slight chance to get that plugin working for textfields in a tabular form? where i can not use the HTML expression property to surround the column data with a span tag

**Comment by martinbnielsen** [comment]
Yes, if you make the column a type "Plain text" in the tabular form, you can still supply the "HTML Expression". It works fine:
https://apex.oracle.com/pls/apex/f?p=42601:2

Brgds
Martin

**Comment by pettinger** [comment]
i see your point. as soon as i change the column typ it is rendered read only, and the plugin works.
But i would like the column to remain editable. you think thats possible?

**Comment by martinbnielsen** [comment]
Hm, I am not sure that is even possible, since the text will be inside a "value" tag of an input item.
I have not seen any solutions like that in other applications, so sorry - you have to find another solution.
